kiliautoml/models/_base_model.py

```diff
@@ -119,7 +135,7 @@ def is_compatible(self, cdt_requested: ModelConditionsRequested) -> bool:
         if strict_conditions and tools_ok:
             # We then check the loose conditions
             self._check_compatible(cdt_requested.ml_backend, self.possible_ml_backend, "ml_backend")
-            self._check_compatible(cdt_requested.model_name, self.advised_model_names, "model_name")
+            self._check_compatible_model(cdt_requested.model_name)
```

Comment:
I propose the following UX: we should still advise models. In this case:
* we outputting a warning saying that the model is not in the advised list.
* provide a CLI command that lists the advised models for a given Kili project.
WDYT?

Reply:
Yes, that could be really nice. I will work on that.